.github/workflows/cd-pypi.yml

```
@@ -58,7 +56,7 @@ jobs:
           fetch-depth: 0 # Disable shallow clones to improve sonarcloud analysis
 
       - name: Build SDist and wheel
-        run: pipx run build
+        run: uvx --from build pyproject-build
```

This command doesn't work, try with 'uv build'
```suggestion
        run: uv build 
```